tests/test-gpio.js

```diff
@@ -21,7 +26,7 @@ var mark = 0;
 
 var edgeInterval = setInterval(function () {
     var count = 0;
-    pinA = gpio.open({ pin: pins.IO7 });
+    pinA = gpio.open({pin: pins.IO7});
```

Comment:
Just following this since I noticed it:

https://google.github.io/styleguide/javascriptguide.xml#Code_formatting
`var obj = {a: 1, b: 2, c: 3};  // No space after { or before }.`